File: data/src/main/java/com/platdmit/data/api/models/ApiDomainRecord.java
package com.platdmit.data.api.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ApiDomainRecord {
    @SerializedName("id")
    @Expose
    private long id;
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("data")
    @Expose
    private String data;
    @SerializedName("priority")
    @Expose
    private int priority;
    @SerializedName("ttl")
    @Expose
    private int ttl;
    @SerializedName("port")
    @Expose
    private int port;
    @SerializedName("weight")
    @Expose
    private int weight;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public int getPriority() {
        return priority;
    }

    public void setPriority(int priority) {
        this.priority = priority;
    }

    public int getTtl() {
        return ttl;
    }

    public void setTtl(int ttl) {
        this.ttl = ttl;
    }

    public int getPort() {
        return port;
    }

    public void setPort(int port) {
        this.port = port;
    }

    public int getWeight() {
        return weight;
    }

    public void setWeight(int weight) {
        this.weight = weight;
    }
}

File: feature_domains/src/main/java/com/platdmit/feature_domains/data/retrofit/models/ApiDomainRecord.kt
package com.platdmit.feature_domains.data.retrofit.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ApiDomainRecord(
        @SerializedName("id")
        @Expose
        val id: Long,

        @SerializedName("type")
        @Expose
        val type: String,

        @SerializedName("name")
        @Expose
        val name: String,

        @SerializedName("data")
        @Expose
        val data: String,

        @SerializedName("priority")
        @Expose
        val priority: Int,

        @SerializedName("ttl")
        @Expose
        val ttl: Int,

        @SerializedName("port")
        @Expose
        val port: Int,

        @SerializedName("weight")
        @Expose
        val weight: Int
)